### PR TITLE
verifydialogs: Pass a unique Key to the create_report

### PR DESCRIPTION
**Pull request by fernflower**
This final touch to stable id generation will make reports from
the same actor but with different dialogs have different ids.

Depends-On: https://github.com/oamg/leapp/pull/669

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/checkhacluster/tests/test_check_ha_cluster_checkhacluster.py [hunk 1]
```diff
@@ -16,7 +16,9 @@ def test_no_inhibit_when_no_ha_cluster(monkeypatch, current_actor_context):
 
 
 def test_inhibits_when_cluster_node(monkeypatch, current_actor_context):
-    monkeypatch.setattr("os.path.isfile", lambda path: True)
+    # NOTE(ivasilev) Limiting here the paths to mock not to cause unexpected side-effects
+    # (original test had path: True)
+    monkeypatch.setattr("os.path.isfile", lambda path: path in [COROSYNC_CONF_LOCATION, CIB_LOCATION])
```
Comment (vinzenz) [inline repos/system_upgrade/el7toel8/actors/checkhacluster/tests/test_check_ha_cluster_checkhacluster.py:21]: nit, I usually prefer tuples for constants like this. Lists are more expensive. No need to change, I just wanted to point this out.

**Comment by AloisMahdal** [comment]
leapp-ci build

**Comment by AloisMahdal** [comment]
leapp-ci build

**Comment by pirat89** [comment]
@fernflower pls add the description to the PR so it's clear even to people out of our team what and why is in this PR.

**Comment by pirat89** [comment]
leapp-ci build

**Comment by AloisMahdal** [comment]
latest **murphy-ci/*** failures are caused by internal bug (my bad -- sorry) which is fixed now; re-running

**Comment by AloisMahdal** [comment]
...on a more positive note: I've extended the test case where "answer-less" `leapp upgrade` is expected to inhibit.

We now expect exact key `d35f6c6b1b1fa6924ef442e3670d90fa92f0d54b` in all of these cases.

--

The new test has *passed* on all supported architectures with:

 *  *leapp-0.11.0-0.202101111435Z.04c5ef0.stablereportid.PR669.el7_8*
 *  *leapp-repository-0.12.0-0.202101121832Z.5cf1bc4.stablereport.PR618.el7_8*

so from QE POV, this PR is good to go.

**Comment by AloisMahdal** [comment]
*murphy-ci/integration* - I have confirmed that the fail can be waived safely.

is: from QE POV, this PR is *good to go*.

**Comment by vojtechsokol** [comment]
leapp-ci build

**Comment by vojtechsokol** [comment]
e2e tests

**Comment by fernflower** [comment]
e2e tests

**Comment by pirat89** [comment]
@fernflower I have rebased the branch apply your fix for travis.

**Comment by pirat89** [comment]
omg. the e2e crashed again.. so running tests before the merge

**Comment by pirat89** [comment]
leapp-ci build

**Review by pirat89 — APPROVED** [review]

**Pull request merged**